hack/util.sh

```diff
@@ -41,7 +41,7 @@ capz::util::should_build_kubernetes() {
 
 # all test regions must support AvailabilityZones
 capz::util::get_random_region() {
-    local REGIONS=("eastus" "eastus2" "northeurope" "uksouth" "westeurope" "westus2")
+    local REGIONS=("eastus" "eastus2" "northeurope" "uksouth" "westeurope" "westus2", "centralus", "canadacentral", "westus3")
```

Comment:
TODO: order alphabetically

Reply:
does alphabetic order help with our randomize func?

Reply:
no it's just a code readability preference